File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--updating-the-controls-on-a-ribbon-at-run-time_7.cs
        void button_Click(object sender, RibbonControlEventArgs e)
        {
            Outlook.Application application = Globals.ThisAddIn.Application;
            Outlook.Inspector inspector = application.ActiveInspector();
            Outlook.MailItem myMailItem = (Outlook.MailItem)inspector.CurrentItem;
            RibbonButton myCheckBox = (RibbonButton)sender;
            myMailItem.Subject = "Following up on your order";
            myMailItem.Body = myMailItem.Body + "\n" + "* " + myCheckBox.Label;
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--updating-the-controls-on-a-ribbon-at-run-time_7.vb
    Private Sub Button_Click(ByVal sender As System.Object, _
        ByVal e As Microsoft.Office.Tools.Ribbon.RibbonControlEventArgs)
        Dim application As Outlook.Application = Globals.ThisAddIn.Application
        Dim inspector As Outlook.Inspector = application.ActiveInspector()
        Dim myMailItem As Outlook.MailItem = CType(inspector.CurrentItem,  _
            Outlook.MailItem)
        Dim myButton As RibbonButton = CType(sender, RibbonButton)
        myMailItem.Subject = "Following up on your order"
        myMailItem.Body = myMailItem.Body + ControlChars.Lf + "* " _
            + myButton.Label
    End Sub